### PR TITLE
Change vbom.ml/util dependeny with github.com/fvbommel/util

### PR DESCRIPTION
**Pull request by rhrazdil**
Added vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
to go.mod.

Signed-off-by: Radim Hrazdil <rhrazdil@redhat.com>

**What this PR does / why we need it**:
CI fails to get vbom.ml/util dependency
```
go mod tidy
go mod vendor
go: k8s.io/kubernetes@v0.0.0-00010101000000-000000000000 requires
	vbom.ml/util@v0.0.0-20160121211510-db5cfe13f5cc: unrecognized import path "vbom.ml/util": https fetch: Get "https://vbom.ml/util?go-get=1": dial tcp: lookup vbom.ml on 127.0.0.1:53: no such host
go: k8s.io/kubernetes@v0.0.0-00010101000000-000000000000 requires
	vbom.ml/util@v0.0.0-20160121211510-db5cfe13f5cc: unrecognized import path "vbom.ml/util": https fetch: Get "https://vbom.ml/util?go-get=1": dial tcp: lookup vbom.ml on 127.0.0.1:53: no such host
make: *** [Makefile:95: vendor] Error 1
```
Same issue https://github.com/kubernetes/kubectl/issues/925 and corresponding fix kubernetes/kubernetes#94451 suggests to replace the dep with github.com/fvbommel/util

Source of the used workaround:
https://github.com/kubevirt/kubevirt/blob/0a731e884b91d236103ed9c1b5e40db849dfd471/hack/builder/vbom_ml.diff

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: vendor/modules.txt
Comment (oshoval) [inline vendor/modules.txt:224]: Please revert all the line reorder, for some reason some local machine reorder,
just keep the real updated lines

**Review by oshoval — CHANGES_REQUESTED** [review]

**Comment by oshoval** [comment]
I would suggest to add the reason (what was the error of vbom and so on to the description) please,
also you can reference the issue on vbom

File: vendor/modules.txt
Comment (rhrazdil) [reply vendor/modules.txt:224]: So I should only add the line in go.mod and do not attempt running `make check`, right?

Comment (oshoval) [reply vendor/modules.txt:224]: I am used to this from kubevirt, there i do run the `make` targets that are needed, in order to find other changes in case there are, but revert the reordering, this what i would suggest to do here as well imo
best ask Miguel

Comment (phoracek) [reply vendor/modules.txt:224]: Why is this file important to be reviewed. It is just generated. And how is the order important?

Comment (oshoval) [reply vendor/modules.txt:224]: it will cause conflicts for others, if anybody would change it as his own machine reorder,
some local machines does changes to the order while some doesn't.

Comment (phoracek) [reply vendor/modules.txt:224]: Understood. There are no others.

Comment (rhrazdil) [reply vendor/modules.txt:224]: If `make format` yield different results every time, it kinda breaks the point of having automated formatter, doesn't it.
I would imagine that we leave the formatting on the tool.

Comment (phoracek) [reply vendor/modules.txt:224]: You're right Radim. And if we get different results, we should invest in fixing of the tools we use.

I don't see a reason to do anything special about the order in this PR. I think it is good to go once you update commit message and description of the PR as Or suggested in https://github.com/kubevirt/macvtap-cni/pull/25#issuecomment-688752807

Comment (oshoval) [reply vendor/modules.txt:224]: until we fix it , we will see this file changes again and again imo

Comment (phoracek) [reply vendor/modules.txt:224]: It is a generated file and there is no need to look at it during reviews.

**Comment by rhrazdil** [comment]
@oshoval @phoracek Improved the description and commit message. Please take another look

**Comment by maiqueb** [comment]
/approve
/lgtm
/hold

Holding so @phoracek can have a final word on this.

**Comment by phoracek** [comment]
We should address https://github.com/kubevirt/macvtap-cni/pull/25#issuecomment-688786560 causing do-not-merge/invalid-commit-message

**Comment by phoracek** [comment]
Looks good, thanks!

/hold cancel

File: vendor/modules.txt
Comment (EdDev) [reply vendor/modules.txt:224]: > It is a generated file and there is no need to look at it during reviews.

If a generated file got changed, I interpret it as something was changed that relates to it. It may cause me to check what it is and if that change is correct or not.
Asking the reviewer to ignore changes of any file is a risk of missing unwanted/unintended changes.
Moreover, if this changes every time one builds the project locally and the file is not ignored by git, it requires the developer to drop the changes manually in order to move between branches.

If this file is changing randomly, do not add it to the tracked files or fix the build to reflect nothing changed.

Obviously, this is a general note, it is not a must to block this change on this, but it will be a good idea to fix this sooner then later.

p.s: I do not like generated code being tracked, it seems to exist all over in k8s related projects. I heard no convincing reasoning for it so far. But it is too generic to discuss it on any specific PR, sorry for the noise.

Comment (oshoval) [reply vendor/modules.txt:224]: in the past when i had it, i found an issue on github about the ordering, worth to try to find it again whoever try to fix this imo.

**Pull request merged**
